MapboxNavigation/NavigationMapView.swift

```diff
@@ -13,6 +14,17 @@ open class NavigationMapView: MGLMapView {
     let sourceCasingIdentifier = "routeCasingSource"
     let routeLayerIdentifier = "routeLayer"
     let routeLayerCasingIdentifier = "routeLayerCasing"
+    let waypointSourceIdentifier = "waypointsSource"
+    let waypointCircleIdentifier = "waypointsCircle"
+    let waypointSymbolIdentifier = "waypointsSymbol"
+    let arrowSourceIdentifier = "arrowSource"
+    let arrowSourceStrokeIdentifier = "arrowSourceStroke"
+    let arrowLayerIdentifier = "arrowLayer"
+    let arrowSymbolLayerIdentifier = "arrowSymbolLayer"
+    let arrowLayerStrokeIdentifier = "arrowStrokeLayer"
+    let arrowCasingSymbolLayerIdentifier = "arrowCasingSymbolLayer"
+    let arrowSymbolSourceIdentifier = "arrowSymbolSource"
+    let isCurrentLeg = "isCurrentLeg"
```

Comment:
https://github.com/mapbox/mapbox-navigation-ios/pull/270#discussion_r134897473 got collapsed for some reason: the name `isCurrentLeg` implies that it’s a Boolean. Since this string is being used as an attribute name, call it `currentLegAttribute` but give it the value `isCurrentLeg`.